README.md

```diff
@@ -65,83 +65,68 @@ For more information and instructions, see the Kubernetes blog post, [Kubernetes
 
 ##### 4.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
-
 <br/>
 <br/>
 <br/>
 
-
 ## Installation
 
-### Install the operator
+### Installing the driver with GitHub
 
+The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
 
-#### 1. Download the manifest from GitHub.
+Use the following steps to install the operator and driver, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
 
-```bash
-curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
-```
+**Note:** Before you begin, you may need to create a user-defined namespace. Create the project namespace, using the `kubectl create ns <namespace>` command.
 
-#### 2. (Optional): If required, update the image fields in the ibm-block-csi-operator.yaml.
-**Note**: Updating the namespace to a user-defined namespace may be necessary in order to ensure consistency and avoid trouble installing the operator.
+#### 1.  Install the operator.
 
-#### 3. Create a namespace.
+##### 1. Download the manifest from GitHub.
 
-```bash
-$ kubectl create ns <namespace>
-```
+	```
```

Comment:
I think this breaks the `` ``` `` markdown
![image](https://user-images.githubusercontent.com/57443811/121533310-d65e8580-ca08-11eb-92a2-3dc781d19f73.png)

Reply:
look like it, done

Reply:
note the formatting and how it's indented here: https://github.com/IBM/ibm-block-csi-driver/blob/documentation/docs/csi_ug_install_operator_github.md. this helps with the substeps